sdk/template/template/karma.conf.js

```diff
@@ -92,10 +92,14 @@ module.exports = function(config) {
     // enable / disable watching file and executing tests whenever any file changes
     autoWatch: false,
 
-    // start these browsers
-    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    // 'ChromeHeadless', 'Chrome', 'Firefox', 'Edge', 'IE'
-    browsers: ["ChromeHeadless"],
+    // --no-sandbox allows our tests to run in Linux without having to change the system.
+    browsers: ["ChromeHeadlessNoSandbox"],
+    customLaunchers: {
+      ChromeHeadlessNoSandbox: {
```

Comment:
I did not require this change when I ran storage tests on Linux a while ago.
(Evidently, our nightly tests also run fine)

Reply:
I have needed to use this change in some versions of Debian.

Reply:
- Can you verify when this is actually needed (maybe with keyvault) ?
- And cross-check with storage-tests to know if this solution is for all packages or if we need to do this selectively

Reply:
This issue is a common problem for running browsers on limited versions of Linux, like how one would run our tests in Docker, for example. Here's one StackOverflow mention I was able to find from a quick search: https://stackoverflow.com/a/45846909/371538

Some projects with considerable popularity do this by default, to allow people to run Chrome Headless on Docker, for example: https://github.com/Zenika/alpine-chrome

Puppeteer has instructions in their documentation on how to configure a usable sandbox on Linux: https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#setting-up-chrome-linux-sandbox

In my opinion, this process is complex and unnecessary, since we do absolutely trust that our tests are safe to run locally.

Reply:
I'm going to change this code snippet to:

```js
  browsers: ["ChromeHeadlessNoSandbox"],
  customLaunchers: {
    ChromeHeadlessNoSandbox: {
        base: "ChromeHeadless",
        // In order to protect the host environment from untrusted web content,
        // Chrome uses multiple layers of sandboxing.
        // For this to work properly, the host should be configured first.
        // If there's no good sandbox for Chrome to use, it will crash with the error `No usable sandbox!`.
        // We absolutely trust that our tests are safe to run locally, in a browser without sandbox.
        flags: ["--no-sandbox"]
    }
  }
```

For now, until we device a better solution. If we happen to agree that this is fine, then we will have saved some time up front.